outbound.js

```diff
@@ -187,6 +211,18 @@
         return deferred.promise;
     };
 
+    Outbound.prototype.trackBatch = function(events) {
```

Comment:
These events need to be validated in the same way the `track` method validates. Each event should have a `user_id` and a `event`. Also, the total size of the array can not exceed 100 events.